### DISCUSSION TITLE
[Auth] Improve pattern matching for provider logo

### DISCUSSION THREAD

**Discussion by Sytten** [ente-io/ente#1860] [Enhancements]
Right now the algorithm seems pretty basic, the name needs to match exactly for the logo to show. It would be nicer if it had a bit of flexibility on it. If I type `Bitwarden ASDF` I still want to see the bitwarden logo.

**Comment by SpeeterYT** [comment]
Can only agree, especially for services which provide such names (Fritz!Box) for example (The names are "FritzBox ####" so FritzBox 7590, FritzBox 7530...)

**Comment by ua741** [comment]
In v4.2.0, you can pick the icon manually.
